### PR TITLE
Introduce JJ for kubernetes-incubator/descheduler

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

File: sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+parent: 'common/test_cases/minimal.yml'
+extensions:
+  actions:
+    - type: "script"
+      title: "run descheduler tests"
+      script: |-
+        cd /data/src/github.com/kubernetes-incubator/descheduler
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml]: So the job is going to do nothing right now, yeah?

File: sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
+---
+parent: 'common/test_cases/minimal.yml'
+extensions:
+  actions:
+    - type: "host_script"
+      title: "Copy GCE credentials into AWS instance"
+      script: |-
+        scp -F ./.config/origin-ci-tool/inventory/.ssh_config /var/lib/jenkins/.config/gcloud/descheduler-credentials.json openshiftdevel:/data/gce-credentials.json
```
Comment (ingvagabund) [inline sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml]: As we discussed

File: sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      title: "run descheduler tests"
+      script: |-
+        export GOPATH=/data
+        export PATH=$GOPATH:$PATH
+        cd $GOPATH/src/github.com/kubernetes-incubator/descheduler
+        make test-e2e
+  system_journals:
+    - systemd-journald.service
+overrides:
+  post_actions:
```
Comment (ingvagabund) [inline sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml]: Well yeah. I don't see any other way. We need to change the GCE project cause we switch to `openshift-descheduler-infra` and if anything goes wrong we need to make sure the GCE project is back to `openshift-gce-devel`. And we need to collect artefacts before the GCE output is assembled. Right now we don't generated any artefacts but later on we will.

**Comment by ingvagabund** [comment]
@stevekuznetsov PTAL

File: sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml (hunk 3 above)
Comment (stevekuznetsov) [reply sjb/config/test_cases/ci-kubernetes-descheduler-e2e-gce.yml]: Not the worst thing in the world, this will probably work.

**Pull request merged**